NewForce-Cohort-6/tabloid-cli-coven-of-the-lunar-owls
# Ticket #18 - Edit Post

# Description

This PR is for [Ticket #18](https://trello.com/c/R5PWPejn), which should allow the user to navigate to Post Management from the Main Menu and then update a post from the list of posts using option 4 of the Post Menu. This has been tested in the CLI and appears to correctly display alterations made, by the user, to the name and/or URL of a preexisting post.  

# Testing Instructions

1. git fetch --all
2. git checkout ac-updatePosts to switch to my branch.
If you don't see the most recent version of the files, try git pull origin ac-updatePosts.
3. Execute TabloidCLI from inside of Visual Studio
4. Select option 4, Post Management, from Main Menu.
5. Select option 4, Edit Post, from Post Menu.
6. Select a post to update from the list of posts.
7. Enter new values where prompted to do so.
8. Select option 1, List Posts, from the Post Management menu and view your changes.




# Checklist:
